catalog.html

```diff
@@ -1,171 +1,190 @@
               </div>
             </div>
-            <div>
-              <p>Рейтинг: 8,5</p>
+            <div class="sort-rezult-right">
+              <p class="sort-rezult-popular">Рейтинг: 8,5</p>
             </div>
-          </article>
-           <article>
-            <div>
+          </div>
+        </article>
+        <article class="sort-rezult-item">
+          <div class="container">
+            <div class="sort-rezult-left">
               <img src="img/card-2.jpg" alt="Desert Quail Inn" width="135" height="90">
-              <h3><a href="#!">Desert Quail Inn</a></h3>
-              <p>Мотель <span>от 3000 Р.</span></p>
-              <div>
+              <h3 class="sort-rezult-title"><a href="#!">Desert Quail Inn</a></h3>
+              <p class="sort-rezult-desc">Мотель <span>от 3000 Р.</span></p>
+              <div class="sort-rezult-button">
                 <a href="#!">Подробнее</a>
                 <button type="button">Забронировать</button>
               </div>
             </div>
-            <div>
-              <p>Рейтинг: 8,3</p>
+            <div class="sort-rezult-right">
+              <p class="sort-rezult-popular">Рейтинг: 8,3</p>
             </div>
-          </article>
-           <article>
-            <div>
+          </div>
+        </article>
+        <article class="sort-rezult-item">
+          <div class="container">
+            <div class="sort-rezult-left">
               <img src="img/card-3.jpg" alt="Villas at Poco Diablo" width="135" height="90">
-              <h3><a href="#!">Villas at Poco Diablo</a></h3>
-              <p>Апартаменты <span>от 3000 Р.</span></p>
-              <div>
+              <h3 class="sort-rezult-title"><a href="#!">Villas at Poco Diablo</a></h3>
+              <p class="sort-rezult-desc">Апартаменты <span>от 3000 Р.</span></p>
+              <div class="sort-rezult-button">
                 <a href="#!">Подробнее</a>
                 <button type="button">Забронировать</button>
               </div>
             </div>
-            <div>
-              <p>Рейтинг: 9,2</p>
+            <div class="sort-rezult-right">
+              <p class="sort-rezult-popular">Рейтинг: 9,2</p>
             </div>
-          </article>
-        </div>
-      </section>
-    </main>
-    <footer>
-      <div>
-        <a href="">#visitSEDONA</a>
-        ТЕЛ: <a href="tel:+74956660166">+7(495) 666-01-66</a>
+          </div>
+        </article>
       </div>
-      <ul>
-       <li>
-        <a href="">
-          <img src="img/twitter.svg" alt="twitter" width="46" height="48">
-        </a>
-      </li>
-      <li>
-        <a href="">
-          <img src="img/fb-icon.svg" alt="facebook" width="46" height="48">
-        </a>
-      </li>
-    </ul>
-    <a href="https://htmlacademy.ru/intensive/htmlcss" target="_blank">
-      <img src="img/logo-htmlacademy.svg" alt="htmlacademy" width="115" height="41">
+    </section>
+  </main>
+  <footer class="main-footer container">
```

Comment:
не обязательно писать main. Можно проще
```suggestion
  <footer class="footer container">
```